### PR TITLE
Story/vspc 93 2

### PR DESCRIPTION
**Pull request by ashmichheda**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
(you can copy the ticket if it hasn't changed) 

...Put description here...
  
## Anything else the reviewer needs to know?

... describe here ...

Renamed the branch `VSPC-93` to `VSPC-93-2` following the GitHub guidelines.

### CODE REVIEW

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Please move the background image and menu icon closer to the top menu bar (I know you didn't change it, but might as well fix it here). Make the menu background white or light grey or something so it fits with the rest of the layout (have Adhvik help you with the design) and make the the font in the menu a little smaller.

**Comment by jdamerow** [comment]
top part needs to be filled out.

**Comment by ashmichheda** [comment]
Which top part? I didn't get you

**Comment by jdamerow** [comment]
top part needs to be completely filled out.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
- image and menu need to be closer to top menu bar
- on mobile, especially when I use my phone landscape, the closing x for the menu is so far up it's almost not possible to click; the menu still doesn't stack above the background iamge but let's leave it the way it is and just fix the closing icon issue.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
- Slides in modules are too close to the menu bar now

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
The menu icon and image need to be closer to the top. The module page is fine but for spaces, the menu icon should be in the left top corner.

**Comment by jdamerow** [comment]
Same as before, the module page should stay where it is and not moved up.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.html [hunk 1]
```diff
@@ -66,26 +81,66 @@
...
-		        $("#mySidenav").css("width", "240px");
-		        $("#mySidenav").css("height", "[(${display.height})]px");
-            }
+		function openNav(){	
+			document.getElementById("mySidenav").style.width = "250px";
         }
-		function closeNav(){
-		    $("#mySidenav").css("width", "0px");
+        function closeNav(){
+        	document.getElementById("mySidenav").style.width = "0";
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.html:143]: Is there a reason you replaced the jquery with regular javascript?

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.html [hunk 2]
```diff
@@ -120,6 +175,7 @@
 		$(window).on("load", function() {
 		    let spaceHeight = $("#bgImage").css("height");
 		    $("#Module_1").css("height",spaceHeight);
+		    $("#Module_1").css("margin", "-45px 0px 60px auto");
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.html:178]: This makes the background image move up after loading; it should be set correctly right away (no jumping up effect).

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.html (hunk 1 above)
Comment (ashmichheda) [reply vspace/src/main/webapp/WEB-INF/views/exhibition/space.html:143]: This code is not added by me. Maybe while indentation and code formatting changes, it appears to have been added by me.

Comment (jdamerow) [reply vspace/src/main/webapp/WEB-INF/views/exhibition/space.html:143]: a) it shouldn't show up as changed and b) clean code rule #X, leave the code better than you find it. ;)

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
The image does not seem to be closer to the top. The space heading should be more or less aligned with the menu icon.
image

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
For future commits, your commit messages should contain the ticket number, not the branch name.

**Pull request merged**